Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/AccountController.java

```diff
@@ -0,0 +1,75 @@
+package controllers;
+
+import dao.UserDAOReal;
+import model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import service.UserService;
+
+import javax.servlet.http.HttpSession;
+
+import java.util.logging.Logger;
+
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+@Controller
+public class AccountController {
+    private static Logger log = Logger.getLogger(AccountController.class.getName());
+
+    @Autowired
+    UserService userService;
+    @Autowired
+    UserDAOReal userDAO;
+
+
+    @RequestMapping(method = GET, value = "/auth")
+    public String getAuthorization() {
+        return "Authorization";
+    }
+
+    @RequestMapping(method = POST, value = "/auth")
+    public String postAuthorization(@RequestParam("login") String login,
+                                    @RequestParam("password") String password,
+                                    HttpSession session) {
+        log.info("Current login:" + login + " Current password:" + password);
+        if ((login != null || password != null) && (userService.authorizate(login,password))) {
```

Comment:
Authorizate is not appropriate name. Login and password null checks are wrong

Reply:
first logical operation will be deleted. 
Can it be possible name to method for example: "isRightAuthData" ?